### ISSUE TITLE
 Problems accessing customerproxy addresses

### ISSUE THREAD

**Issue by jhonvidal** [awesto/django-shop#819]
> It's `admin:shop_customerproxy_changelist`
> 
> Small hint to find out yourself:
> 
> * Go to the admin page you are looking for.
> * Copy the URL without the site part, here `/en/admin/shop/customerproxy/`.
> * Run `./manage.py shell`
> * Use that URL in function `resolve`.
> * Recheck with `reverse` if the URL is the same.
> 
> ```
> >>> from django.urls import *
> >>> resolve('/en/admin/shop/customerproxy/')
> ResolverMatch(func=django.contrib.admin.options.changelist_view, args=(), kwargs={}, url_name=shop_customerproxy_changelist, app_names=['admin'], namespaces=['admin'])
> >>> reverse('admin:shop_customerproxy_changelist')
> '/en/admin/shop/customerproxy/'
> ```

Regarding that URL, I have problems accessing the following addresses:

http://localhost:8000/es/admin/shop/customerproxy/4/change/
or
http://localhost:8000/es/admin/shop/customerproxy/add/

I have the following error:

```
TypeError at /es/admin/shop/customerproxy/4/change/

has_add_permission() takes 2 positional arguments but 3 were given

Request Method: 	GET
Request URL: 	http://localhost:8000/es/admin/shop/customerproxy/4/change/
Django Version: 	3.0.7
Exception Type: 	TypeError
Exception Value: 	

has_add_permission() takes 2 positional arguments but 3 were given

Exception Location: 	/home/ikki/.virtualenvs/vmedexample/lib/python3.6/site-packages/django/contrib/admin/options.py in get_inline_instances, line 596
Python Executable: 	/home/ikki/.virtualenvs/vmedexample/bin/python
Python Version: 	3.6.0
Python Path: 	

['/home/ikki/Desarrollo/implementacion/medexample',
 '/home/ikki/Desarrollo/implementacion/medexample',
 '/home/ikki/Desarrollo/ide/pycharm-2020.1.1/plugins/python/helpers/pycharm_display',
 '/home/ikki/.pyenv/versions/3.6.0/lib/python36.zip',
 '/home/ikki/.pyenv/versions/3.6.0/lib/python3.6',
 '/home/ikki/.pyenv/versions/3.6.0/lib/python3.6/lib-dynload',
 '/home/ikki/.virtualenvs/vmedexample/lib/python3.6/site-packages',
 '/home/ikki/Desarrollo/ide/pycharm-2020.1.1/plugins/python/helpers/pycharm_matplotlib_backend']

Server time: 	Lun, 15 Jun 2020 16:33:36 +0000
```

I have made from the installation  `cookiecutter-django-shop`  by default with all the sample data.

what is the solution?

_Originally posted by @jhonvidal in https://github.com/awesto/django-shop/issues/785#issuecomment-644244405_

**Comment by jrief** [comment]
Thanks for reporting.
Fixed in version 1.2.2

**Comment by jhonvidal** [comment]
Thank you!
